_test/test_sqw/test_mask.m

```diff
@@ -23,15 +31,28 @@
         % Swallow any warnings for when pixel page size set too small
         obj.old_warn_state = warning('OFF', 'PIXELDATA:validate_mem_alloc');
 
+        % 2D case setup
         obj.sqw_2d = sqw(obj.sqw_2d_file_path);
-        obj.sqw_2d_paged = obj.get_paged_sqw();
+        obj.sqw_2d_paged = obj.get_paged_sqw(obj.sqw_2d_file_path);
 
         obj.idxs_to_mask = [2, 46, 91, 93, 94, 107, 123, 166];
         obj.mask_array_2d = ones(size(obj.sqw_2d.data.npix), 'logical');
         obj.mask_array_2d(obj.idxs_to_mask) = 0;
 
         obj.masked_2d = mask(obj.sqw_2d, obj.mask_array_2d);
         obj.masked_2d_paged = mask(obj.sqw_2d_paged, obj.mask_array_2d);
+
+        % 3D case setup
```

Comment:
is there any reason this isn't split into two separate cases: `test_mask_filters_correct_pixels_with_2d_sqw_object` and `test_mask_filters_correct_pixels_with_3d_sqw_object`?

Reply:
Not sure what you mean. Do you mean two separate test classes?

Reply:
Ignore this. Misread the collapsed code.